decompiler/pipeline/dataflowanalysis/dead_loop_elimination.py

```diff
@@ -26,7 +28,7 @@ class DeadLoopElimination(DeadPathElimination, PipelineStage):
 
     def __init__(self):
         """Initialize a new loop elimination."""
-        self._logic_converter: Z3Converter = Z3Converter()
+        self._logic_converter: BaseConverter = DelogicConverter()
```

Comment:
One minor question, is there a good reason to choose `DelogicConverter` as default? I think I would prefer that z3-is the fallback when the config option is, for example, misspelled or what do you think?

Reply:
I think that makes sense. Let me make the corresponding changes!